### PR TITLE
Android dark mode

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] init app background color
- [x] set app background color on change
- [x] test 28
- [ ] prod build

### CODE REVIEW

**Comment by MarcoPolo** [comment]
It currently relies on JS to make the initial call to Android to set the initial background color. It would probably be better to make a Go FFI that lets java ask for the current setting at startup.

Should that be a part of this PR?

**Comment by MarcoPolo** [comment]
I'm also unclear why we save the dark mode preference right after we get it.

**Comment by chrisnojima** [comment]
👍 merging once ci is green

File: shared/android/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -11,4 +11,5 @@
     </style>
 
     <color name="white">#ffffff</color>
+    <color name="black">#000000</color>
```
Comment (hyperobject) [inline shared/android/app/src/main/res/values/styles.xml]: this needs to be #191919 to match dark mode colors

**Pull request merged**